cmake/BOTG.cmake

```diff
@@ -190,6 +203,7 @@ MACRO( botgLibrary name )
         ELSE()
             MESSAGE( FATAL_ERROR "[BootsOnTheGround] botgLibrary LANGUAGE ${libraryLanguage} is unknown (Fortran|C|CXX)")
         ENDIF()
+        SET_TARGET_PROPERTIES( "${name}" PROPERTIES LINKER_LANGUAGE "${library_LANGUAGE}" )
```

Comment:
CMake Error at external/BootsOnTheGround/cmake/BOTG.cmake:206 (SET_TARGET_PROPERTIES):
  set_target_properties Can not find target to add properties to:
  Testing123_Fortran
Call Stack (most recent call first):
  external/Testing123/src/CMakeLists.txt:52 (botgLibrary)